features/users/authenticate_users.feature

```diff
@@ -105,6 +105,16 @@ Feature: User Authentication
       And 1 email should be delivered to "notsam@example.com"
       And the email should have "Reset your password" in the subject
       And the email to "notsam" should be non-translated
+    When 1 email should be delivered to "sam@example.com"
```

Comment:
Shouldn't this step be a `Then` (well technically an `And` because of the above steps)